src/api.ts

```diff
@@ -647,7 +648,7 @@ export async function createContainer( imageName: ImageName, env: RunEnv ) {
 			id: container.id,
 		} );
 	} catch ( error ) {
-		l.error( { imageName, error }, `Failed creating container for ${ imageName }` );
+		error.message = `Failed creating container for ${ imageName }: ` + error.message;
```

Comment:
Nit: it feels a little awkward to use both a template literal and string concatenation in a single expression like this.